README.md

```diff
@@ -48,93 +48,93 @@ For each CODATA release, we provide two maps; one for the data values and one fo
+  | Planck constant over 2 pi         | `hbar`                | 1.054 571 800 e-34   | 0.000 000 013 e-34   | J s            |
+  | Rydberg constant                  | `rydberg`             | 10 973 731.568 508   | 0.000 065            | m^-1           |
+  | Speed of light in a vacuum        | `c`                   | 2.997 924 58  e8     | 0.0                  | m s^-1         |
+  | Speed of light in a vacuum        | `speedOfLight`        | 2.997 924 58  e8     | 0.0                  | m s^-1         |
+  | Stefan-Boltzmann constant         | `stefanBoltzmann`     | 5.670 367 e-8        | 0.000 013 e-8        | W m^-2 K^-4    |
 
 #### Particle masses
- | Quantity            | Variable name  | Value                 | Uncertainty           | Unit   | 
- | --------            | -------------  | ----                  | ------                | ------ | 
- | electron mass       | `electronMass` | 9.109 383 7015 e-31   | 0.000 000 0028 e-31   | kg     | 
- | neutron mass        | `neutronMass`  | 1.674 927 498 04 e-27 | 0.000 000 000 95 e-27 | kg     | 
- | proton mass         | `protonMass`   | 1.672 621 923 69 e-27 | 0.000 000 021 e-27    | kg     | 
- | deuteron mass       | `deuteronMass` | 3.343 583 7724 e-27   | 0.000 000 0010 e-27   | kg     | 
- | helion mass         | `helionMass`   | 5.006 412 7796 e-27   | 0.000 000 0015 e-27   | kg     | 
- | triton mass         | `tritonMass`   | 5.007 356 7446 e-27   | 0.000 000 0015 e-27   | kg     | 
- | alpha particle mass | `alphaMass`    | 6.644 657 3357 e-27   | 0.000 000 0020 e-27   | kg     | 
+ | Quantity            | Variable name  | Value                 | Uncertainty           | Unit   |
+ | --------            | -------------  | ----                  | ------                | ------ |
+ | electron mass       | `electronMass` | 9.109 383 7015 e-31   | 0.000 000 0028 e-31   | kg     |
+ | neutron mass        | `neutronMass`  | 1.674 927 498 04 e-27 | 0.000 000 000 95 e-27 | kg     |
+ | proton mass         | `protonMass`   | 1.672 621 923 69 e-27 | 0.000 000 021 e-27    | kg     |
+ | deuteron mass       | `deuteronMass` | 3.343 583 7724 e-27   | 0.000 000 0010 e-27   | kg     |
+ | helion mass         | `helionMass`   | 5.006 412 7796 e-27   | 0.000 000 0015 e-27   | kg     |
+ | triton mass         | `tritonMass`   | 5.007 356 7446 e-27   | 0.000 000 0015 e-27   | kg     |
+ | alpha particle mass | `alphaMass`    | 6.644 657 3357 e-27   | 0.000 000 0020 e-27   | kg     |
 
 
 ## Derived Constants
 Some constants are derived from others, but used sufficiently often that it is worth pre-computing the values. Their values are not listed here, but the variable names are.
 
-| Quanitity      | Variable Name  | Unit | 
-| ----           | -----          | ---- | 
-| 2π             | `twopi`        | --   | 
-| Reduced Planck | `hbar`         | J s  | 
-| Planck length  | `planckLength` | m    | 
+| Quantity       | Variable Name  | Unit |
```

Comment:
This is a good catch!

Reply:
OK, I went and did the small fixups in metaconfigure/ instead; you should see a PR there.